releases/release-1.31/release-notes/maps/pr-126330-map.yaml

```
@@ -0,0 +1,4 @@
+pr: 126330
+releasenote:
+  text: Built kubernetes with `Go 1.22.5`.
```

To match `releases/release-1.31/release-notes/maps/pr-126047-map.yaml`, I suggest:
```suggestion
  text: Updated kubernetes to build with Go 1.22.5.
```